packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/server.ts

```diff
@@ -43,18 +35,23 @@ export function getServerConfig(wco: WebpackConfigOptions) {
   };
 
   if (wco.buildOptions.bundleDependencies == 'none') {
+    const preserveSymlinks = !!wco.buildOptions.preserveSymlinks;
     config.externals = [
       /^@angular/,
-      // tslint:disable-next-line:no-any
-      (_: any, request: any, callback: (error?: any, result?: any) => void) => {
+      (context: string, request: string, callback: (error?: null, result?: string) => void) => {
         // Absolute & Relative paths are not externals
         if (request.match(/^\.{0,2}\//)) {
           return callback();
         }
 
         try {
           // Attempt to resolve the module via Node
-          const e = require.resolve(request);
+          // we cannot use require.resolve, because it has preserveSymlinks logic
+          const e = resolve(request, {
```

Comment:
We can actually use `require.resolve` here.  If execution got here and the request can be resolved then it should always be an external.  The conditionals below this that test for node_modules can be removed.  All builtins are also externals for the node target.

Reply:
updated